app/src/androidTest/java/com/github/epfl/meili/auth/FirebaseAuthenticationServiceTest.kt

```diff
@@ -43,6 +46,11 @@ class FirebaseAuthenticationServiceTest {
         }
     }
 
+    @After
+    fun after() {
+        FirebaseAuthenticationService.authProvider = { Firebase.auth }
+    }
+
```

Comment:
Why is this file being changed?

Reply:
Because there was interaction with the google sign in test which made it fail so this fixes it